database/seeder/generator.js

```diff
@@ -70,6 +56,8 @@ const generateBiz = () => {
     const type = [];
     type.push(_.sample(typeOne));
     type.push(_.sample(typeTwo));
+    let nhood = '';
+    nhood += _.sample(district);
     const bizname = name;
     const reviewCount = faker.random.number(8000);
     const rating = faker.random.arrayElement([1, 1.5, 2, 2.5, 3, 3.5, 4, 4.5, 5]);
```

Comment:
While the ratings are shown in the business info, the Reviews service will have an actual api that returns a review rating for the end product.(Same with the review count) The general reason we wouldn't want to keep reviewCounts or ratings in the business database is to keep things DRY. (Don't Repeat Yourself) I imagine if a review is updated, added or deleted, the business logic to update and keep track of changes in both the Biz and Reviews db would double.

Reply:
I totally forgot this part...lol I thought I only need to create user schema for testing maybe I should create a review schema as well. But ....I think this time I will wait til I am able to test your API. Otherwise, I probably spent too much time on this.